### PR TITLE
 [WEEX-86][doc] update the docs of the scroller component and navigator module, make the description more understandable.

### PR DESCRIPTION
**Pull request by boboning**
* add some examples to show how use some new attributes and event for scroller.
* update the example of the navigator module.
* make the description more understandable for above docs.

### CODE REVIEW

File: source/references/modules/navigator.md [hunk 1]
```diff
@@ -42,7 +41,8 @@ pop a weex page onto the navigator stack, you can specify whether apply animatio
 ```html
 <template>
```
Comment (Hanks10100) [inline source/references/modules/navigator.md]: Documents should not contain so many source codes.

File: source/references/components/scroller.md [hunk 2]
```diff
@@ -1,40 +1,39 @@
...
+order: 2.8
 version: 2.1
 ---
 
 # &lt;scroller&gt;
 
 <span class="weex-version">v0.6.1+</span>
 
-A scroller is a component in vertical direction which can have multiple child components in one column. If total height of its child components exceed the height of the scroller, the whole child components will be scrollable.
+Scroller is a component which can have multiple child components in one column. It supports both direcitons. If the content size of child components exceeds the frame of the scroller, the whole child components will be scrollable.
```
Comment (cxfeng1) [inline source/references/components/scroller.md]: Maybe the description should be more reasonable, please refer to [iOS UIScrollView](https://developer.apple.com/documentation/uikit/uiscrollview?language=objc#) or [Android ScrollView](https://developer.android.com/reference/android/widget/ScrollView.html)

File: source/references/components/scroller.md [hunk 3]
```diff
@@ -1,40 +1,39 @@
...
 
 ## Child Components
 
 Scroller supports all kinds of components, such as div, text, etc.
-And there are two special components that can only be used inside scroller component.
+There are two special components that can only be used inside scroller component.
 
-* refresh 0.6.1 used inside list to add pull-down-to-refresh functionality.
-* loading 0.6.1 used inside list to add pull-up-to-load-more functionality.
+* refresh 0.6.1 used inside list to add pull-down-to-refresh functionality.
```
Comment (cxfeng1) [inline source/references/components/scroller.md]: Please use version tag to mark supported version.

File: source/references/components/scroller.md [hunk 4]
```diff
@@ -1,40 +1,39 @@
...
 ## Child Components
 
 Scroller supports all kinds of components, such as div, text, etc.
-And there are two special components that can only be used inside scroller component.
+There are two special components that can only be used inside scroller component.
 
-* refresh 0.6.1 used inside list to add pull-down-to-refresh functionality.
-* loading 0.6.1 used inside list to add pull-up-to-load-more functionality.
+* refresh 0.6.1 used inside list to add pull-down-to-refresh functionality. 
+* loading 0.6.1 used inside list to add pull-up-to-load-more functionality.
```
Comment (cxfeng1) [inline source/references/components/scroller.md]: used inside scroller

File: source/references/components/scroller.md [hunk 5]
```diff
@@ -1,40 +1,39 @@
...
-* loading 0.6.1 used inside list to add pull-up-to-load-more functionality.
+* refresh 0.6.1 used inside list to add pull-down-to-refresh functionality. 
+* loading 0.6.1 used inside list to add pull-up-to-load-more functionality. 
 
 
 ## Attributes
 
-* show-scrollbar: true/false whether show the scroll bar or not, default value is true
-* scroll-direction: <string> define scroll direction of component, horizontal or vertical
+* show-scrollbar: &lt;boolean&gt;  true | false, default value is true. This attribute indicates whether show the scroll bar or not.
```
Comment (cxfeng1) [inline source/references/components/scroller.md]: A Boolean value that controls whether the scroll indicator is visible.

File: source/references/components/scroller.md [hunk 6]
```diff
@@ -153,4 +153,4 @@ For example, a vertical list nested in a vertical list or scroller is not allowe
 </style>
 ```
 
-[try it](http://dotwe.org/vue/2f22f14fb711d88515e63c3f67bed46a)
+[try it](http://dotwe.org/vue/2f22f14fb711d88515e63c3f67bed46a)
```
Comment (cxfeng1) [inline source/references/components/scroller.md]: Maybe we should add more examples here.

File: source/references/modules/navigator.md [hunk 7]
```diff
@@ -1,17 +1,16 @@
...
 ---
 
-# Navigator
+# Navigator 
 
 <span class="weex-version">v0.6.1+</span>
 
 As it's known to all that, we can navigate back and forth in the web browser using the navigation bar.
-And The navigator module mimics the same behaviors in the iOS/Android application. Without such an ability, We will have to stay in the same page forever, so it is very important. Besides the navigation, the module can let us to specify whether to apply animation or not during the transition.
+And The navigator module mimics the same behaviors in the iOS/Android application. Without such an ability, We will have to stay in the same page forever, so it is very important. Besides the navigation, the module can specify whether to apply animation or not during the transition.
```
Comment (cxfeng1) [inline source/references/modules/navigator.md]: This description does not make any sense...

Comment (cxfeng1) [reply source/references/modules/navigator.md]: And maybe it is reasonable to relate to navigator adapter.

**Pull request closed without merging**